src/Bicep.Core/CoreResources.resx

```diff
@@ -286,4 +286,15 @@
   <data name="MaxNumberResourcesRuleMessageFormat" xml:space="preserve">
     <value>Too many resources. Number of resources is limited to {0}.</value>
   </data>
+  <data name="PreferBarePropertyNamesDeclarationFixTitle" xml:space="preserve">
+    <value>Replace quoted property name with {0}.</value>

```

Comment:
<img width="395" alt="image" src="https://user-images.githubusercontent.com/6913354/171232232-681bb8e2-307f-4ba9-a346-619948537131.png">

```suggestion
    <value>Replace quoted property name with '{0}'</value>
```

@ucheNkadiCode Good?

Reply:
Rethinking it, why are they called "property names" @jeskew and @StephenWeatherford ? Couldn't we get away with "UsePropertyNoQuotationFixTitle" and "Remove quotes from property" ?

Reply:
There's some ambiguity as to whether that refers to the property name or the property value, though it is pretty clear from the green squiggly line what the warning is referring to. Any preference, @StephenWeatherford?

Reply:
For me, property & property value is more clear than property name & property value. 
I get confused with "one of the property names in the `obj` object is `myProp`" instead of "one of the properties of `obj` is `myProp`" would love to hear Stephen's opinion on this

Reply:
Suggest we go with best of both worlds:
  <value>Remove quotes from property name '{0}'.</value>